src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataResource.java

```diff
@@ -82,6 +87,21 @@ ResponseEntity updateCaseData(@PathVariable UUID caseUUID, @PathVariable UUID st
         return ResponseEntity.ok().build();
     }
 
+    @Authorised(accessLevel = AccessLevel.WRITE)
+    @PutMapping(value = "/case/{caseUUID}/stage/{stageUUID}/extension")
```

Comment:
I think this should be a POST as we are create a brand new extension.

Reply:
I have made this change